docs/jax201.md

```diff
@@ -0,0 +1,55 @@
+# Jax in Practice
+
+A great way to learn advanced JAX usage is to see how other libraries are using JAX, both how they integrate the library into their API, what functionality it adds mathematically, and how it's used for computational speedup in other libraries.
+
+JAX can be used as a standalone pattern, where the user defines all the calculations themselves. An example is provided here where JAX is entirely used to define the mathematical operations neural network architecture and loss function, and then the computational speedup to estimate the network parameters.
+
+Below we break down various ways JAX is used and provide examples that you can learn from.
+
+## Composable Architecture
+JAX is quite composable and can be used in multiple ways.
+Below we show various examples of how JAX can be incorporated in your
+workflow or library.
+
+### Direct Usage
+Jax can be directly imported and utilized to build models “from scratch” as shown across this website,
+for example in [JAX 101](https://jax.readthedocs.io/en/latest/jax-101/index.html)
+or [Neural Network with JAX](https://jax.readthedocs.io/en/latest/notebooks/neural_network_with_tfds_data.html)
+
+### Model specific Libraries with JAX exposed
+Another common approach is to create a package that provides clean APIs for prepackaged models. An example of this architecture is [Dynamax](https://github.com/probml/dynamax),
+where state space models are already defined and users do some light specification and plug in their own values,
+or [Flax](https://github.com/google/flax) which simplifies the construction Neural Networks.
+
+### Composable Functionality
+Libraries within the JAX ecosystem are often mixed and matched.
+For example [Flax](https://github.com/google/flax) and [Optax](https://github.com/deepmind/optax) are commonly used together, where Flax defines the neural network architecture.
```

Comment:
"... and Optax extends its optimization & model-fitting capabilities"?

Reply:
Added!